app_dart/test/request_handlers/github/webhook_subscription_test.dart

```diff
@@ -1692,6 +1692,11 @@ void foo() {
             ..deletionsCount = 20
             ..additionsCount = 0
             ..changesCount = 20,
+          PullRequestFile()
+            ..filename = 'flutter/lib/ui/bar.java'
+            ..deletionsCount = 20
+            ..additionsCount = 0
+            ..changesCount = 20,
```

Comment:
It seems more meaningful if we have a separate unit test with only `.java` file without `.dart` to validate the updated logic.